src/app/start-page/start-page.scss

```diff
@@ -0,0 +1,110 @@
+$salmon: rgb(255, 82, 82);
+$brown: rgb(116, 32, 6);
+
+html {
+  height: 100%;
+}
+
+.img {
+  background-image: url("../../resources/start-page.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100%;
+}
+
+body {
+  height: 100%;
+}
+
+.md-button {
+  &.md-fab {
+    background-color: $salmon;
+    &:not([disabled]):hover {
+      background-color: rgb(197, 65, 65);
+    }
+  }
+}
+
+h1,
+.reg {
+  padding-top: 10%;
+  text-shadow: 1px 1px 2px black;
+}
+
+.salmon {
+  color: $salmon;
+}
+
+.skyblue {
+  color: skyblue;
+}
+
+.brown {
+  color: $brown;
+}
+
+.container {
+  height: 35px;
+  width: 350px;
+}
+
+md-input-container:not(.md-input-invalid) .md-input {
+  color: $salmon;
+}
+md-input-container:not(.md-input-invalid).md-input-focused .md-input {
+  border-color: $salmon;
+}
+md-input-container:not(.md-input-invalid) .md-input {
+  border-color: rgb(172, 172, 172);
+}
+md-input-container:not(.md-input-invalid).md-input-focused label {
+  color: $salmon;
+}
+md-input-container:not(.md-input-invalid) .md-input label {
+  color: skyblue;
+}
+
+.container-fluid {
+  padding-top: 10%;
+}
+
+li {
+  height: 15%;
+  list-style-type: none;
+  text-align: center;
+  border: 2px solid $salmon;
+  background-color: $salmon;
+  color: white;
+  box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.26);
+  margin-bottom: 10px;
+  margin-top: 10px;
+  margin-right: 25px;
+  margin-left: 25px;
```

Comment:
Instead of do this in four lines you can do it like this:
`margin: 10px 25px;`  (margin: <verticalMargin>, <horizontalMargin>)
You jhave this override too: 
`margin: <top>, <right>, <bottom>, <left>`